website/docs/r/vapp_static_routing.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_static_routing"
+sidebar_current: "docs-vcd-resource-vapp-static-routing"
+description: |-
+  Provides a vCloud Director vApp static routing resource. This can be used to create, modify, and delete static routing rules.
+---
+
+# vcd\_vapp\_static\_routing
+
+Provides a vCloud Director vApp static routing resource. This can be used to create,
+modify, and delete static routing rules in a [vApp network](/docs/providers/vcd/r/vapp_network.html).
+
+~> **Note:** Resource used for vApp network, not vApp Org network.
+!> **Warning:** Using this resource overrides any existing static routing rules on vApp network. It's recommended to have only one resource per vApp. 
+
+## Example Usage
+
+```hcl
+resource "vcd_vapp" "web" {
+  name = "web"
+}
+​
+resource "vcd_vapp_network" "vapp-net" {
+  name               = "my-vapp-net"
+  vapp_name          = vcd_vapp.web.name
+  org_network_name   = "my-vdc-int-net"
+  gateway            = "192.168.2.1"
+  netmask            = "255.255.255.0"
+  dns1               = "192.168.2.1"
+
+  firewall_enabled = true
+  nat_enabled      = true
+​
+  static_ip_pool {
+    start_address = "192.168.2.51"
+    end_address   = "192.168.2.100"
+  }
+}
+
+resource "vcd_vapp_static_routing" "vapp1_static_routing" {
+  vapp_id    = vcd_vapp.web.id
+  network_id = vcd_vapp_network.vapp-net.id
+  enable     = false
+
+  rule {
+    name         = "drule1"
+    network_cidr = "10.10.0.0/24"
+    next_hop_ip  = "192.168.2.2"
+  }
+
+  rule {
+    name         = "rule2"
+    network_cidr = "10.10.1.0/24"
+    next_hop_ip  = "192.168.2.3"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations.
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level.
+* `vapp_id` - (Required) The identifier of [vApp](/docs/providers/vcd/r/vapp.html).
+* `network_id` - (Required) The identifier of [vApp network](/docs/providers/vcd/r/vapp_network.html).
+* `enable_ip_masquerade` - (Optional) Enable or disable static Routing. Default value is `true`.
```

Comment:
Confused. In the example we have `enable`, which is not documented (and should be called `enabled`), but then we have this funny `enable_ip_masquerade`, which description doesn't match the one from the NAT rules:

<img width="1177" alt="Screenshot 2020-06-23 at 13 40 00" src="https://user-images.githubusercontent.com/11330689/85394306-0b06d200-b557-11ea-9f4d-4f234c521d09.png">

Copy/paste flop?

Reply:
fixed, just left from copy.